src/pages/postgraphile/relations.md

```diff
@@ -80,7 +84,7 @@ create table a.post (
       body
 
       # this relation is automatically exposed
-      personByAuthorId {
+      author {
```

Comment:
I don't think this example uses the simplify inflectors plugin since it has `allPosts` at the top.

```suggestion
      personByAuthorId {
```